client/app/pages/home/home.html

```diff
@@ -1,5 +1,8 @@
 <div class="container">
-  <div ng-if="!$ctrl.isEmailVerified" class="alert alert-warning">
+  <div ng-if="$ctrl.messages.includes('using-deprecated-embed-feature')" class="alert alert-warning">
+      You have enabled <code>ALLOW_PARAMETERS_IN_EMBEDS</code> in <code>redash/settings/__init__.py</code>. This setting is now deprecated and supported by default and can be turned off. <a href="https://discuss.redash.io/t/support-for-parameters-in-embedded-visualizations/3337" target="_blank">Read more</a>.
```

Comment:
```suggestion
      You have enabled <code>ALLOW_PARAMETERS_IN_EMBEDS</code>. This setting is now deprecated as parameters in embeds supported by default and should be turned off. <a href="https://discuss.redash.io/t/support-for-parameters-in-embedded-visualizations/3337" target="_blank">Read more</a>.
```

Maybe we should mention something about string parameters?

Reply:
A bit of a problematic way to phrase it. 
> This setting is now deprecated as parameters in embeds *are* supported by default and should be turned off.

Sounds a bit like "parameters in embeds" should be turned off.

Reply:
> Maybe we should mention something about string parameters?

Too detailed and subject to change IMO. Also that's mentioned in the forum post.